charts/shiny-app/templates/fluentd-configmap.yml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: "{{ .Chart.Name }}"
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+  name: "{{ template "fullname" . }}-fluentd"
+data:
+  elasticsearch_host: "{{ .Values.Elasticsearch.Host }}"
```

Comment:
Ah! Just noticed you pushed more commits after I pulled your branch!
I see these values in the environment config files.